r/README.md

```diff
@@ -310,12 +310,9 @@ Other applications of `arrow` are described in the following vignettes:
 -   `vignette("arrow", package = "arrow")`: access and manipulate Arrow
     objects through low-level bindings to the C++ library
 
-## Getting help
+The Arrow for R [cheatsheet](https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow.pdf) and [Cookbook](https://arrow.apache.org/cookbook/r/index.html) are additional resources for getting started with `arrow`.
```

Comment:
I wonder if `https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow.pdf` is the right URL here. Ideally, we would have something like `apache.arrow.com/r/cheatsheet.pdf` — though I bet we need to wait for a release to have that happen. At the very least, if we use `-` instead of `master` we will be redirected to the right branch if/when we change the default branch to `main` `https://github.com/apache/arrow/blob/-/r/cheatsheet/arrow.pdf`

Reply:
@jonkeane I updated it to reflect the last change from @stephhazlitt

Reply:
@pachadotdev I think the suggestion is to change`https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow.pdf` changed to `https://github.com/apache/arrow/blob/-/r/cheatsheet/arrow.pdf` for now.

Reply:
@pachadotdev I just updated this url to keep the PR moving.